schemas/vertices/gene_reaction_complexes.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Groups of genes that take part in producing a chemical reaction in the cell.",
+  "required": ["_key", "conjunctions"],
+  "properties": {
+    "_key": {
+      "type": "string",
+      "description": "Hash of the conjunctions."
+    },
+    "conjunctions": {
+      "type": "array",
+      "example": [["A", "B"], ["C", "D"]],
+      "description": "Array of conjunctions of genes. Eg: (A AND B) OR (C AND D).",
```

Comment:
I thought we were going to define these as just "AND" nodes and represent OR by having multiple complexes? I think that would make the traversal easier and it a bit closer to biological reality.

Reply:
I think I just misremembered it. Yeah the complexes are just a single flat array of AND-ed genes.